.github/workflows/iroha2-dev.yml

```
@@ -7,6 +7,10 @@ on:
 env:
   CARGO_TERM_COLOR: always
 
+concurrency:
+  group: I2::Dev::Deploy-${{ github.workflow }}-${{ github.ref }}
```

`I2::Dev::Deploy` and `${{ github.workflow }}` are duplicates.

```suggestion
  group: ${{ github.workflow }}-${{ github.ref }}
```

The same for other `*.yml`